M2/Macaulay2/m2/localring.m2

```diff
@@ -33,7 +33,7 @@ localRing = method(TypicalValue => LocalRing)
        describe LocalRing := RP -> Describe (expression localRing) (expression ring RP.MaximalIdeal, expression max RP)
      expression LocalRing := RP -> if hasAttribute(RP, ReverseDictionary) then expression getAttribute(RP, ReverseDictionary) else describe RP
 toExternalString LocalRing:= RP -> toString describe RP
-coefficientRing LocalRing := RP -> coefficientRing ring RP.MaximalIdeal
+coefficientRing LocalRing := RP -> frac coefficientRing ring RP.MaximalIdeal
```

Comment:
On the other hand, `coefficientRing localRing(ZZ[x], ideal x)` should be `QQ`, shouldn't it? Not sure what it the right thing here.

Reply:
Here's what I'll do: I'll leave this as is, but document `(coefficientRing, LocalRing)` with a new caveat that the result may be unexpected if the generators of the maximal ideal are contained in the base ring, and that this may be improved in the future.

Reply:
I'll revert this and open a separate issue to correct this.